diff --git a/node_modules/apollo-server-integration-cloudflare-workers/dist/kv-cache.d.ts b/node_modules/apollo-server-integration-cloudflare-workers/dist/kv-cache.d.ts
index eb5b3d4..e257782 100644
--- a/node_modules/apollo-server-integration-cloudflare-workers/dist/kv-cache.d.ts
+++ b/node_modules/apollo-server-integration-cloudflare-workers/dist/kv-cache.d.ts
@@ -1,5 +1,6 @@
 import type { KeyValueCacheSetOptions, KeyValueCache } from "@apollo/utils.keyvaluecache";
 export declare class KVCache implements KeyValueCache {
+    constructor(env: any);
     get(key: string): Promise<string | undefined>;
     set(key: string, value: string, options?: KeyValueCacheSetOptions): Promise<void>;
     delete(key: string): Promise<void>;
diff --git a/node_modules/apollo-server-integration-cloudflare-workers/dist/kv-cache.js b/node_modules/apollo-server-integration-cloudflare-workers/dist/kv-cache.js
index c19822c..dba676f 100644
--- a/node_modules/apollo-server-integration-cloudflare-workers/dist/kv-cache.js
+++ b/node_modules/apollo-server-integration-cloudflare-workers/dist/kv-cache.js
@@ -1,6 +1,10 @@
 export class KVCache {
+    env;
+    constructor(env) {
+        this.env = env;
+    }
     async get(key) {
-        const value = await GRAPHQL_CACHE.get(key);
+        const value = await this.env.GRAPHQL_CACHE.get(key);
         return value === null ? undefined : value;
     }
     set(key, value, options) {
@@ -8,10 +12,10 @@ export class KVCache {
         if (options) {
             opts.expirationTtl = Number(options.ttl) < 60 ? Number(options.ttl) : undefined;
         }
-        return GRAPHQL_CACHE.put(key, value, opts);
+        return this.env.GRAPHQL_CACHE.put(key, value, opts);
     }
     delete(key) {
-        return GRAPHQL_CACHE.delete(key);
+        return this.env.GRAPHQL_CACHE.delete(key);
     }
 }
 //# sourceMappingURL=kv-cache.js.map
\ No newline at end of file
diff --git a/node_modules/apollo-server-integration-cloudflare-workers/src/kv-cache.ts b/node_modules/apollo-server-integration-cloudflare-workers/src/kv-cache.ts
index 9dca0e0..ae1df79 100644
--- a/node_modules/apollo-server-integration-cloudflare-workers/src/kv-cache.ts
+++ b/node_modules/apollo-server-integration-cloudflare-workers/src/kv-cache.ts
@@ -1,9 +1,13 @@
 import type { KeyValueCacheSetOptions, KeyValueCache } from "@apollo/utils.keyvaluecache";
-import type { KVNamespacePutOptions } from "@cloudflare/workers-types";
+import type { KVNamespacePutOption } from "@cloudflare/workers-types";
 
 export class KVCache implements KeyValueCache {
+  env: any;
+  constructor(env: any) {
+    this.env = env;
+  }
   async get(key: string) {
-    const value = await GRAPHQL_CACHE.get(key);
+    const value = await this.env.GRAPHQL_CACHE.get(key);
     return value === null ? undefined : value;
   }
 
@@ -14,10 +18,10 @@ export class KVCache implements KeyValueCache {
       opts.expirationTtl = Number(options.ttl) < 60 ? Number(options.ttl) : undefined;
     }
 
-    return GRAPHQL_CACHE.put(key, value, opts);
+    return this.env.GRAPHQL_CACHE.put(key, value, opts);
   }
 
   delete(key: string) {
-    return GRAPHQL_CACHE.delete(key);
+    return this.env.GRAPHQL_CACHE.delete(key);
   }
 }
